src/proof/nova.rs

```diff
@@ -1472,72 +1472,70 @@ mod tests {
 
     #[test]
     #[ignore]
-    fn outer_prove_evaluate_zero_arg_lambda1() {
+    fn outer_prove_evaluate_zero_arg_lambda() {
         let s = &mut Store::<Fr>::default();
         let expected = s.num(123);
         let terminal = s.get_cont_terminal();
         nova_test_aux(
             s,
-            "((lambda () 123))",
+            "((lambda(x) x) 123)",
```

Comment:
This completely changes the meaning of the test. The previous was indeed testing application of a zero-arg lambda when called with zero arguments. The change makes it a test of a one-arg lambda called with one argument. These are not equivalent, and I think the former is what is intended (and needed).